DataReader/src/main/resources/application.properties

```diff
@@ -1,2 +1,6 @@
 spring.application.name=IMInboundPipeline
 server.port=9082
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.url=jdbc:postgresql://localhost:5432/discovery
+spring.datasource.username=postgres
+spring.datasource.password=123456
```

Comment:
are these being used anywhere?

Connection manager is expecting the values to be populated as env vars and defaults to another static value if it's not

  private static final String POSTGRES_PASSWORD = Optional.ofNullable(System.getenv("POSTGRES_PASSWORD")).orElse("123456");